samples/servicecontrol/retry-messages/Core_9/Receiver/SimpleMessageHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+public class SimpleMessageHandler :
+    IHandleMessages<SimpleMessage>
+{
+    static ILog log = LogManager.GetLogger<SimpleMessageHandler>();
```

Comment:
I don't think we should use `NServiceBus.Logging` in new samples and should use `ILogger`